### DISCUSSION TITLE
desktop.github.com está offline

### DISCUSSION THREAD

**Discussion by luizg99** [community/community#145309] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

desktop.github.com está offline e não consigo acessar para fazer o download para windowns.

**Comment by shinybrightstar** [comment]
Hi @luizg99 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
